### ISSUE TITLE
Objects not seen in the field of view

### ISSUE THREAD

**Issue by Billuc** [KitwareMedical/SlicerVirtualReality#129]
Hi,

Before anything, I am a newbie with Slicer !
I am testing this extension as we want to use VR more.
I managed to follow the installation process, opened a project, clicked on the button to toggle VR display ... and nothing appeared... I got a blue background and that was it.

We fiddled  a bit and found out that some display modes (stereo modes), made the scene "blurrier", as if there was a semi-transparent object in front of us (kinda like a ghost object). However, we never could get a clear view of the objects we wanted to see.

For information, here is some information about the setup we are using:
OS : Windows 10
Slicer version : 5.2.2
Extension version : 2023-05-12
VR headset : Meta Quest 2 (it is not in the list of supported headset, but since it worked with SteamVR, I wanted to give it a shot)

Is there something wrong with my setup ? Like my Slicer version is too old ? A parameter I may have not set correctly ? Or is the plugin not meant to work with the Quest 2 (yet ?) ?
Also, do you have any idea of what could cause the "blur" ? 

Thank you in advance for any information :)
Luc B.

**Comment by cpinter** [comment]
Your Slicer version is definitely too old. As a general rule, we cannot help enyone with an error report who does not use the latest Slicer, because we only have a chance to fix anything in the latest Slicer.

VR especially has been quite hectic in the past years. Now it works more or less well, however, please be mindful that there have been reports of problems with it on the forum regarding interactions.
https://discourse.slicer.org/

Also, although I think I have some power of imagination and was trying hard based on your description, but without a screenshot it is practically impossible to know what you saw.

Anyway, bottom line: please try the latest Slicer, and use the forum. Here on GitHub we discuss confirmed bugs and features we are working on. Thank you!

**Comment by Billuc** [comment]
Thanks for the quick reply !

I will try with the latest version of Slicer then ! I haven't thought of the forums, thanks for pointing me to that !
Hahaha yeah, my description wasn't very clear either ^^' sorry about that 

Still, thank you very much, and I will use the forum next time ;)
